code/modules/surgery/organs/lungs.dm

```diff
@@ -228,21 +278,21 @@
 
```

Comment:
Wonder if I should remove line 273: `if(abs(310.15 - breath.temperature) > 50)` for even more variety of lungs...